src/armature/Node.ts

```diff
@@ -27,12 +27,17 @@ export class Node {
             vec3.fromValues(0, 0, 1),
             vec3.fromValues(1, 0, 0)
         ],
-        indices: [0, 1, 2, 0, 2, 3, 0, 3, 4, 5, 2, 1, 5, 3, 2, 5, 4, 3],
-        colors: Node.boneVertices.map(() => vec3.fromValues(1, 1, 1))
+        indices: [0, 1, 2, 0, 2, 3, 0, 3, 4, 0, 4, 1, 5, 2, 1, 5, 3, 2, 5, 4, 3, 5, 1, 4],
+
+        // Map x, y, z to r, g, b to give a sense of bone orientation
+        colors: Node.boneVertices.map((v: vec3) =>
+            vec3.fromValues(v[0], v[1] / 0.1 + 0.1, v[2] / 0.1 + 0.1)
```

Comment:
What do these `0.1` constants refer to? Are they just arbitrarily defined?

Reply:
For each vertex, x values go between 0 and 1, y and z go between -0.1 and 0.1, so this is normalizing to get each to be between 0 and 1